package.json

```
@@ -5,15 +5,15 @@
   "main": "index.js",
   "dependencies": {
     "execspawn": "^1.0.1",
-    "minimist": "^1.2.0",
-    "mkdirp": "^0.5.1",
-    "node-abi": "^2.0.0",
-    "npm-run-path": "^3.1.0",
+    "mkdirp-classic": "^0.5.3",
+    "node-abi": "^2.19.1",
+    "npm-run-path": "^3",
```

```suggestion
    "npm-run-path": "^3.1.0",
```

Revert to what we had, prevents npm deduping, has more information, keeps ranges consistent.